about.html

```diff
@@ -0,0 +1,141 @@
+    <div class="navbar">
+        <div class="logo">
+           <a href="index.html"><img src="./images/newlogo.png" alt="African Tech Summit Logo"></a>
+        </div>
+        <nav>
+            <ul>
+                <li><a href="about.html">About</a></li>
+                <li><a href="#">Program</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+                <li class="border-about"><a href="#">CC Campaign</a></li>
+            </ul>
+        </nav>
+    </div>
+    </div>
+        <div class="bars" id="hambug">
+        <div class="hambuger"></div>
+        <div class="hambuger"></div>
+        <div class="hambuger"></div>
+        </div>
+
+        <div class="toggle-menu">
+            <div><img src="../images/togclose.png" alt="close toggle" id="togclose" ></div>
+
+            <div class="hambug-items">
+            <ul class="menu">
+                <li class="list-menu"><a href="about.html" class="a-link">About</a></li>
+                <li class="list-menu"><a href="#" class="a-link">Program</a></li>
+                <li class="list-menu"><a href="#" class="a-link">Join</a></li>
+                <li class="list-menu"><a href="#" class="a-link">Sponsor</a></li>
+                <li class="list-menu"><a href="#" class="a-link">News</a></li>
+                <li class="list-menu"><a href="#" class="a-link">CC Campaign</a></li>
+            </ul>
+            </div>
+
+    </div>
+    </header>
+
+    <section class="hero">
+        <div class="container intro">
+        <!-- <h1>"Hello! Africa"</h1> -->
+        <h2>AFRICA TECH SUMMIT 2022</h2>
+        <P class="description">Africa Tech Summit Nairobi connects tech leaders from the African ecosystem and international players under one roof. Network with key stakeholders including tech corporates, mobile operators, fintechs, DeFi & crypto ventures, investors, leading start-ups, regulators and industry stakeholders driving business and investment forward.</P>
+        
+        <P>Please contact us via Emial for further questions <br> concerning Africa Tech Summit 2022</P>
+
+        <p><span>africatechsummit2022@afsummit.org</span></p>
+        </div>
+    </section>
+
+    <section class="summit-logo">
+        <div class="summit-head">
+            <h2>Africa Tech<br> Summit 2022 Logo</h2>
+            <hr>
+        </div>
+        <p>The logo for Africa Tech Summit 2022 was decided through the logo competition from 8 June to 7 July</p>
+        <div class="afrilogo"><img src="./images/tech-logo.png" alt="Africa Tech summit logo"></div>
+    </section>
+
+    <section class="past-events">
+        <div class="past">
+        <h2>See Past Africa Tech Summits</h2>
+        <hr>
+        </div>
+        <div class="time"><p>Take a look at the last two Africa Tech summit that took place in Kenya and Rwanda</p></div>
+    
+        
+
+        <div class="event-container">
```

Comment:
- [ ] I can see that  you leave some useless new lines that make your code less readable, Please remove all blank lines to make your code more readable 
-  For readability, add two spaces of indentation instead of four, you can check the [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

Reply:
Blank lines have been removed